script/Arcueid/Day 11/11_00_ARC11_2.txt

```diff
@@ -279,7 +279,7 @@ Chasing that woman will only get you killed. You mustn't meet Arcueid Brunestud
 [sha:776dcd98f303f69bb408d0a3bbc1b0194cc87027]{
 -- Page 58, Offset 29660.
 -- 「一度でも吸血衝動に負けた真祖は、もう手遅れだからです。
-"If a True Ancestor succumbs to their vampiric urges even once,
+"If a True Ancestor succumbs to their Vampiric Urges even once,
```

Comment:
did we decide to make this a proper noun? I thought we decided on the opposite

Reply:
Petrikow marked it caps on the glossary, I guess take it up with him
https://tsukihimates.com/glossary/#vampires_vampiric_urge

Reply:
yeah it is in the glossary but not in the spreadsheet so I'm unsure what to do with this really
![image](https://user-images.githubusercontent.com/90473809/186635503-da008020-219b-4c19-ba3c-d9555d857460.png)